src/contracts/interfaces/IEigenPod.sol

```diff
@@ -99,7 +109,7 @@ interface IEigenPod {
     function verifyWithdrawalCredentialsAndBalance(
         uint64 oracleBlockNumber,
         uint40 validatorIndex,
-        BeaconChainProofs.ValidatorFieldsAndBalanceProofs memory proofs,
+        bytes memory proofs,
```

Comment:
is there a good reason for this changing? seems we are still using the same struct type in other functions?

Reply:
Yes, we don't want balance proofs for this function